tutor/specs/components/exercises/__snapshots__/sectionizer.spec.js.snap

```diff
@@ -13,33 +13,23 @@ exports[`Exercise Sectionizer Component renders and matches snapshot 1`] = `
   <div
     className="section"
     onClick={[Function]}
-  >
-    1
```

Comment:
This change seems odd.

Reply:
yes, turned out the spec wasn't using the model like it should